src/style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: center;
+
+  height: 96px;
+  width: 96px;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+  height: 16px;
+  justify-content: space-between;
+  align-content: center;
+}
+
+.stars__star {
+  content: "";
```

Comment:
no need for content, you already use background-image

```suggestion

```

Reply:
also you need to add margin-right: 4px to each star (except the last one in the row) to separate them from one another

Reply:
after you fix everything try to pass the tests again